modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2160,6 +2162,73 @@ test.describe('Manage object entries through View Object Entries', () => {
 			viewObjectEntriesPage.page.getByText('astronaut.png')
 		).toBeVisible();
 	});
+
+	test('verify that an appropriate error message appears after attempting to upload an oversized file', async ({
+		apiHelpers,
+		page,
+		viewObjectEntriesPage,
+	}) => {
+
+		// page.setViewportSize({height: 1080, width: 1920});
+
+		const {objectFields} = await mockObjectFields({
+			apiHelpers,
+			objectFieldBusinessTypes: ['richText'],
+		});
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFields,
+				status: {code: 0},
+			});
+
+		apiHelpers.data.push({
+			id: objectDefinition.id,
+			type: 'objectDefinition',
+		});
+
+		await test.step('Go to the object entry page, click to add an entry, attempt to upload the files, and verify the error messages', async () => {
+			await viewObjectEntriesPage.goto(objectDefinition.className);
+
+			await viewObjectEntriesPage.clickAddObjectEntry(
+				objectDefinition.label['en_US']
+			);
+
+			const filePath = path.join(__dirname, 'dependencies', 'planet.jpg');
+
+			const fileBase64 = fs.readFileSync(filePath).toString('base64');
+
+			const imageHtml = `<p><img alt="" src="data:image/jpeg;base64,${fileBase64}" /></p>`;
+
+			const sourceButton = page.getByLabel('Source');
+
+			await sourceButton.click();
+
+			await page.getByRole('textbox').last().fill(imageHtml);
+
+			await sourceButton.click();
+			await page.getByRole('button', {name: 'Save'}).click();
```

Comment:
can we use `await viewObjectEntriesPage.saveObjectEntryButton.click();` here as well? thanks!